src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskForSearchCompletionControllerCFTTest.java

```diff
@@ -199,6 +201,7 @@ public void should_return_a_200_and_retrieve_single_task_when_one_of_the_task_do
             .contentType(APPLICATION_JSON_VALUE)
             .body("task_required_for_event ", is(false))
             .body("tasks.size()", equalTo(1))
+            .body("tasks[0].permissions.values", hasItems("Read","Refer","Own","Manage","Cancel"))
```

Comment:
for searchForCDomleteable, we should only assert the relevant permissions. OWN and EXECUTE. Applicable to all tests for this type of search.

Reply:
The minimum permissions required for this endpoint is OWN, EXECUTE . If either of those permissions match then we return the record. That record may contain other permissions as well in addition to own, excecute.